AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/models/ReportApiModelBase.java

```diff
@@ -10,6 +10,14 @@
     private String reportBody;
     private int agentId;
 
+    public String getReportTitle() {
```

Comment:
This looks like it shouldn't be here?

Reply:
As in, you removed the field report title so we shouldn't have the getter or setter any more?